packages/schema/test/readability.js

```diff
@@ -86,13 +86,13 @@ describe('readability', () => {
       operation: {
         perform: '$func$2$f$',
         sample: { id: 1 },
-        inputFields: [{ key: 'field', type: 'string', default: '' }],
+        inputFields: [{ key: 'field', type: 'string', children: [] }],
       },
     });
     results.errors.should.have.length(1);
     should(results.errors[0].property.endsWith('instance')).be.false();
     results.errors[0].property.should.eql(
-      'instance.operation.inputFields[0].default',
+      'instance.operation.inputFields[0].children',
```

Comment:
Why the change? Was it failing?

Reply:
It was failing earlier because default was removed and I changed it, passes now with `default` but since they effectively test the same thing I left it as is. I'll change it back so the # of files changes decreases at least!